ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/actions/dependsOnCoralBasicActions.js

```diff
@@ -30,6 +30,15 @@
         return $tabPanel.closest('coral-tabview').find('coral-tablist > coral-tab').eq($tabPanel.index());
     }
 
+    /**
+     * Toggle visibility of every field on the tab
+     */
+    function tabChildrenVisibility($tabPanel, state) {
```

Comment:
To separate PR just in case: It looks like we need to move it to move tabs logic to a separate file.